.travis.yml

```diff
@@ -46,12 +46,14 @@ matrix:
            CHECK_MBSTRING=0
   allow_failures:
     - php: "hhvm"
+    - php: "nightly"
     - php: "master"
+    - php: "7.4"
 
 install:
     - composer self-update
-    - composer install
-    - if [[ $USE_PSALM -eq 1 ]]; then composer require --dev "vimeo/psalm:^0|^1|^2"; fi
+    - if [[ $USE_PSALM -eq 1 ]]; then composer require "vimeo/psalm:^0|^1|^2"; fi
```

Comment:
The `composer require "vimeo/psalm:^0|^1|^2"` will install required dependencies defined on `composer.json` at the same time.

Reply:
Psalm and PHPUnit should never be installed in a production environment. Omitting the `--dev` flag can lead developers who learn from Paragon's code to develop bad habits that can bite them down the line. We would like to avoid that.

Reply:
Agree. Revert the `--dev` option.